openpype/hosts/maya/plugins/publish/extract_playblast.py

```diff
@@ -65,12 +68,44 @@ def process(self, instance):
         # get cameras
         camera = instance.data["review_camera"]
 
-        preset = lib.load_capture_preset(data=self.capture_preset)
-        # Grab capture presets from the project settings
-        capture_presets = self.capture_preset
+        host_name = instance.context.data["hostName"]
+        family = instance.data["family"]
+        task_data = instance.data["anatomyData"].get("task", {})
+        task_name = task_data.get("name")
+        task_type = task_data.get("type")
+        subset = instance.data["subset"]
+
+        filtering_criteria = {
+            "hosts": host_name,
```

Comment:
Does the profile filter criteria require `host`? seems a bit much for a feature we know only exists in Maya nor has any host setting in the profile's settings schema?

Reply:
@iLLiCiTiT ?

Reply:
Have refactored to `lib` and hardcoded host to `maya`.